packages/app-web/src/slices/settings/index.ts

```diff
@@ -25,7 +25,10 @@ const defaultSettingsState: SettingsState = {
     'control+z': undoAction.type,
     'shift+meta+z': redoAction.type,
     'control+shift+z': redoAction.type,
-    'escape': [popModalAction.type, endWireAction.type]
+    'control+y': redoAction.type, // Windows and Linux systems that use Cinnamon as a DE use Ctrl+y for redo. See https://support.apple.com/en-ie/guide/pages/tana7e101d4c/mac, https://support.microsoft.com/en-us/office/undo-redo-or-repeat-an-action-84bdb9bc-4e23-4f06-ba78-f7b893eb2d28, and https://cheatography.com/shakiestnerd/cheat-sheets/linux-mint-cinnamon/
```

Comment:
Move your comment into its own line above this one and wrap it at a line length of 80 characters (without breaking links into two).

Reply:
I moved them onto a new line, but the links are longer than 80 characters, is that fine?

Reply:
There's an 80 characters soft cap and a 120 characters hard cap for the line length. I will add this to the styleguides.